vignettes/wrapping-apis.Rmd

```diff
@@ -281,24 +293,28 @@ So now you can understand how the following code works:
 my_key <- secret_decrypt("4Nx84VPa83dMt3X6bv0fNBlLbv3U4D1kHM76YisKEfpCarBm1UHJHARwJHCFXQSV", "HTTR2_KEY")
 ```
 
+JENNY COMMENT HERE
+
 Since most people won't have this key, n the source of this vignette I also check whether or not it's available with `has_auth <- secret_has_key("HTTR2_KEY")`, then use `knitr::opts_chunk(eval = has_auth)` so that the code is only run when possible.
 
 When `secret_decrypt()` is run in tests, it will automatically skip the test if the key isn't found.
-That means your tests will automatically pass on CRAN.
+That means that your auth-requiring tests will automatically be skipped on CRAN.
 
 ### Security considerations
 
-Note that including an API key in a query is a relatively insecure; if an API uses it, its typically because the key is relatively easy to create or gives relatively few privileges.
+Note that including an API key as a query parameter is relatively insecure; if an API uses this method of auth, it's typically because the key is relatively easy to create or gives relatively few privileges.
 Here it only takes a couple of minutes to generate your own NYTimes API key, so there's little incentive for someone to try and steal yours.
 
-The main problem of storing credentials in the url is that it's easily revealable, because httr2 makes no efforts to redact confidential information stored in query parameters.
+The main problem of conveying credentials via the url is that it's easily exposed, because httr2 makes no efforts to redact confidential information stored in query parameters.
 This means it's relatively easy to leak your key if you use `req_perform(verbose = 1)`, `req_dry_run()`, or even just print the request object.
 
-All in all, this mean you need to be a little cautious if you need to get help with a reprex; you'll need to manually redact the key.
+All in all, this means you need to be a little cautious if you need to get help with a reprex; you'll need to manually redact the key.
 
 ### Basic request
 
-Now lets perform a test request and look at the response:
+Now let's perform a test request and look at the response:
+
+JENNY COMMENT HERE
```

Comment:
Ooops! I'll have to think about how to avoid this? Maybe some `req_redact()` function that's automatically applied to the output of `req_dryrun()`? 

Or maybe I should just embrace it, with a comment.

Reply:
Decided to embrace it